sdk/src/wallet/wallet/builder.rs

```diff
@@ -58,21 +58,22 @@ impl<S: SecretManage> Default for WalletBuilder<S> {
 #[cfg(feature = "storage")]
 #[cfg_attr(docsrs, doc(cfg(feature = "storage")))]
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub(crate) struct StorageOptions {
-    pub(crate) storage_path: PathBuf,
-    pub(crate) storage_file_name: Option<String>,
-    pub(crate) storage_encryption_key: Option<[u8; 32]>,
-    pub(crate) manager_store: ManagerStorage,
+#[serde(rename_all = "camelCase")]
+pub struct StorageOptions {
+    pub(crate) path: PathBuf,
+    pub(crate) file_name: Option<String>,
+    pub(crate) encryption_key: Option<[u8; 32]>,
+    pub(crate) kind: StorageKind,
 }
 
 #[cfg(feature = "storage")]
 impl Default for StorageOptions {
     fn default() -> Self {
         Self {
-            storage_path: default_storage_path().into(),
-            storage_file_name: None,
-            storage_encryption_key: None,
-            manager_store: ManagerStorage::default(),
+            path: default_storage_path().into(),
+            file_name: None,
```

Comment:
This is an unused field we can remove, no?

Reply:
Removed